django-stubs/db/models/fields/generated.pyi

```diff
@@ -11,7 +11,6 @@ from django.utils.datastructures import DictWrapper
 from django.utils.functional import _StrOrPromise
 
 class GeneratedField(models.Field):
-    generated: Literal[True]
```

Comment:
Why not `generated: ClassVar[Literal[True]]`?

Reply:
I thought the value was already defined in the parent class so I didn't need it.
If the values are not the same, do I need to specify them separately?